spec/support/shared/spree_builders.rb

```diff
@@ -1,5 +1,4 @@
 RSpec.shared_context 'spree_builders' do
-  # TODO: This starts already to be out of hand. Think about fixing this
```

Comment:
You can see the issue here:
https://github.com/glossier/solidus_retail/issues/17